lib/arena.rb

```diff
@@ -1,2 +1,56 @@
+# require "pry"
+require_relative 'gladiator'
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize (name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(variable)
+    if @gladiators.length < 2
+      @gladiators << variable
+    end
+  end
+
+  def fight
+    # @gladiators.select! do |x|
+    #     x.name == "Maximus"
+    #   end
+    if @gladiators.length == 2
```

Comment:
I think we talked about this once, but I'd think this could be cut down to just an each loop and using something like i and i+1